src/views/asset-view/layout.tsx

```diff
@@ -47,7 +47,7 @@ const AssetSideBar = ({
                     <TenureDetails tenure={tenure} />
                 </SideBarSection>
             </SideBar>
-            {hasToggle('MMH.CreateTenure') &&
+            {useFeatureToggle('MMH.CreateTenure') &&
```

Comment:
Don't call hooks in the return of the render function.

Reply:
Moved hook out of render function